usermembership/tool/src/java/org/sakaiproject/umem/tool/ui/UserListBean.java

```diff
@@ -115,12 +114,21 @@
 	private String							message				= "";
 	// system
 	/** Sakai services vars */
+	//TODO - SakaiFacade api
```

Comment:
Sorry I don't agree that this is a good way to go. My issue with Facades like this is that there becomes an even larger surface area for people to understand. Before tools like Mockito made it easy to mock out a whole service and just implement the methods you were interested I can understand some reasons for it. But not so much now.

What I think makes things harder is one facade class will have:

    public User getUser(String id) {
        try {
            return userDirectoryService.getUser(id);
        catch( UserNotDefinedException unde) {
            return null;
        }

and another class will have in it's facade:

    public User getUser(String userId) {
                try {
            return userDirectoryService.getUser(id);
        catch( UserNotDefinedException unde) {
            return userDirectoryService.getAnonymousUser();
        }

so when you're in the tool you always have to go into the implementation to check they don't have different handling.

Reply:
OK, 
1) SakaiFacade delete
2) UserListBean: UserTimeService ComponentManager ..